docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md

```diff
@@ -1,18 +1,25 @@
 ---
-title: Sideload Office Add-ins for testing
-description: 'Learn how to sideload an Office add-in for testing'
-ms.date: 03/18/2020
+title: Sideload Office Add-ins for testing from a network share
+description: 'Learn how to sideload an Office add-in for testing from a network share'
+ms.date: 06/02/2020
 localization_priority: Normal
 ---
 
-# Sideload Office Add-ins for testing
+# Sideload Office Add-ins for testing from a network share
+
+You can install an Office Add-in for testing in an Office client running on Windows by publishing the manifest to a network file share (instructions below). This deployment option is primarily intended to be used when you have completed development and testing on a localhost and want to test the add-in from a non-local server or cloud account.
+
+> [!IMPORTANT]
+> Deployment by network share is not supported for production add-ins. If you use this method for a production add-in, note these limitations:
```

Comment:
It feels weird to say "Don't use this is production, but if you do..." Consider this instead:
```suggestion
> Deployment by network share is not supported for production add-ins. This method has the following limitations:
```